AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingViewController.swift

```diff
@@ -390,8 +390,10 @@ extension MeetingViewController: RealtimeObserver {
                 self.currentRoster[attendeeId] = RosterAttendee(
                     name: nil, attendeeId: attendeeId, volume: .notSpeaking, signal: .high)
                 self.dispatchGroup.enter()
+                let encodedTitle = HttpUtils.encodeStrForURL(str: self.meetingId ?? "")
+                let encodedAttendee = HttpUtils.encodeStrForURL(str: attendeeId)
                 HttpUtils.getRequest(
-                    url: "\(AppConfiguration.url)attendee?title=\(self.meetingId ?? "")&attendee=\(attendeeId)",
+                    url: "\(AppConfiguration.url)attendee?title=\(encodedTitle)&attendee=\(encodedAttendee)",
```

Comment:
Do we need to encode each of them or is it possible to do something like 
`HttpUtils.encodeForURL(str: "\(AppConfiguration.url)attendee?title=\(self.meetingId ?? "")&attendee=\(attendeeId)")`?

Reply:
Yep, It should work. Changed and tested.